src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/integration/client/PrisonApiClientTest.kt

```diff
@@ -113,22 +113,36 @@ class PrisonApiClientTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `retrieve agency`() {
-    val request = HttpRequest.request().withPath("/api/agencies/prison/MDI")
+  fun `retrieve agency, including inactive`() {
+    val request = HttpRequest.request().withPath("/api/agencies/MDI%3FactiveOnly=false")
```

Comment:
I think you can use `withQueryStringParameters` here if you want to avoid having to handle the query parameter syntax and encoding.